README.md

```diff
@@ -30,6 +30,8 @@ Company                                           | Business/products
 [Cloudbeds](https://www.cloudbeds.com/)           | Hospitality Software                     | Web, DevOps                                    | ✓                                    | Anywhere                | :keyboard: [Jobs](https://www.cloudbeds.com/careers)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-Cloudbeds-EI_IE1399374.11,20.htm)
 [Cloudbees](https://www.cloudbees.com/)           | CI/CD, Jenkins, Codeship                 | Java, Web, Go, Vuejs, Rails                    | ✓                                    | Anywhere                | :keyboard: [Jobs](https://www.cloudbees.com/careers)<br>:door: [Glassdoor](https://www.glassdoor.ca/Overview/Working-at-CloudBees-EI_IE1106279.11,20.htm)
 [Codelathe](https://www.getfilecloud.com/)        | Enterprise File Sharing, Sync and Backup | PHP, Web, Node, C++                            | ✓                                    | Anywhere                | :keyboard: [Jobs](https://www.getfilecloud.com/jobs)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-FileCloud-EI_IE2893485.11,20.htm) 
+[Coinbase](https://www.coinbase.com/)             | Digital currency wallet and platform     | DevOps, Kinesis, PostgreSQL, Ruby, Go          | 
```

Comment:
Not convinced about Coinbase yet, as they only have a handful of jobs advertised as remote.